.idea/vcs.xml

```diff
@@ -2,6 +2,7 @@
 <project version="4">
   <component name="VcsDirectoryMappings">
     <mapping directory="$PROJECT_DIR$" vcs="Git" />
-    <mapping directory="$PROJECT_DIR$/ivy_tests/test_array_api" vcs="Git" />
```

Comment:
And this?

Reply:
@hello-fri-end, Black formatting made the changes to these files locally yet i reverted the changes.